types/es-abstract/test/helpers/index.ts

```diff
@@ -0,0 +1,26 @@
+import './DefineOwnProperty.test';
```

Comment:
why do we need types for es-abstract's own test files? nobody should be importing them.

Reply:
These files exist so that I don’t have to edit `tsconfig.json` with every release to add new test files to the `files` array: <https://github.com/DefinitelyTyped/DefinitelyTyped/blob/06eb285a3042aabe88c28961cdddff1b83fdf15f/types/es-abstract/tsconfig.json#L15-L18>

Compare that to what’s on the `master` branch: <https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d9f8c5c420b3dcaf7317be4c19b478800f38740d/types/es-abstract/tsconfig.json#L15-L51>

Reply:
can't the files array have `test/**/*.js` or something?

Reply:
No, because [`files`](https://www.typescriptlang.org/v2/en/tsconfig#files) doesn’t use globs and `include` [is disallowed by `types‑publisher`](https://github.com/microsoft/types-publisher/blob/54bd71c9fed3c13fefed2284c3fad62bfac66bdd/src/lib/definition-parser.ts#L288-L290).

Reply:
ah, sad, npm's `files` does use globs.